### ISSUE TITLE
Add Indication of Successful Loop Run

### ISSUE THREAD

**Issue by channemann** [loudnate/openaps-monitor#17]
Some sort of visual cue that the loop ran successfully would be great. I'm imagining a flat red line at the very bottom of the graph that is present whenever the loop completes and breaks whenever it doesn't. This could (perhaps) be accomplished by looking at the timestamp of the last json file in the loop?

**Comment by loudnate** [comment]
I think more importantly, mmhistorytools should have an optional argument to append a successful dose report to history data so the monitor isn't looking at the loop before the last one.

**Comment by channemann** [comment]
Agree, that would give a full log of successful runs then. It would still be great to show it on the plot, even in a very subtle way.
